src/index.md

```diff
@@ -49,7 +49,7 @@ Rust is a large language and we won't be able to cover all of it in a few days.
 Some non-goals of this course are:
 
 - Learning how to develop macros: please see
-  [Chapter 19.5 in the Rust Book](https://doc.rust-lang.org/book/ch19-06-macros.html)
```

Comment:
Rather than try to track an unstable version number here, let's just refer to the Rust Book as a whole.

Reply:
ok, so you will have "_Learning how to develop macros: please see the Rust Book and [Rust by Example](https://doc.rust-lang.org/rust-by-example/macros.html) instead._" 
or?